src/Tribe/Organizer.php

```diff
@@ -310,6 +310,19 @@ public function save( $id, $data, $post_type, $post_status ) {
 	 *
 	 */
 	public function save_meta( $organizerId, $data ) {
+		$organizer = get_post( $organizerId );
+
+		/**
+		 * Allow hooking in prior to updating meta fields.
+		 *
+		 * @param int     $organizerId The organizer ID we are modifying meta for.
+		 * @param array   $data        The meta fields we want saved.
+		 * @param WP_Post $organizer   The organizer itself.
+		 *
+		 * @since TBD
+		 */
+		do_action( 'tribe_events_organizer_save', $organizerId, $data, $organizer );
```

Comment:
No need to change anything, just asking out of curiosity: why an action and not, say, a filter (for `$data`) ... or both? The action is nice in of itself, and could be used as a trigger to setup a further hook to modify things later, but might a filter serve a more practical purpose for more users?

Reply:
I kept it the same as what the same hook logic was for events. There aren't filters yet for event/organizer/venue for the data, they only have actions.

Reply:
I think it would be useful to eventually add filters.

Reply:
Hey guys! 

Just adding a little bit of feedback here as I faced a ticket today where I was looking into a way to change the data before is saved into the meta field, I wasn't able to find any so I think it might be useful to have one (I faced on the `Venue` file). 

The way I handled was by using something like: 

```php 
add_action( 'tribe_community_event_created', function( $event_id ) {
    $venue_id = tribe_get_venue_id( $event_id );
    update_post_meta( $venue_id, '_VenueShowMapLink', 'false' );
    update_post_meta( $venue_id, '_VenueShowMap', 'false' );
});
```

cc @barryhughes @sc0ttkclark

Reply:
I'd be interested in being able to have filters for event/organizer/venue data saving too. I  think that can be something we tackle in the future, or someone can PR it.